atomic

```diff
@@ -683,6 +683,9 @@ def create_parser(help_text):
     verifyp.add_argument("-v", "--verbose", default=False,
                           action="store_true",
                           help=_("Report status of each layer"))
+    verifyp.add_argument("-V", "--validate", default=False,
+                                action="store_true",
+                                help=_("validate the state of an image"))
     return parser
```

Comment:
Right, I just think this might be more important functionality then what we had before.  I would like to see verify grow to 
* verify the image is the lates,
* verify the images is signed accoding to policy
* verify the image has not been tampered with (This patch)